tt_metal/common/test_tiles.hpp

```diff
@@ -105,10 +128,10 @@ inline std::vector<T> untilize_nchw(const BufferType<T>& in, const std::vector<s
     result.resize(batch_size * H * W);
     uint32_t linear = 0;
     for (auto batch_index = 0; batch_index < batch_size; batch_index++) {
-        for (int hs32 = 0; hs32 < H; hs32 += 32) {        // iterate over h with stride 32
-            for (int ws32 = 0; ws32 < W; ws32 += 32) {    // iterate over w with stride 32
-                for (int h32 = 0; h32 < 32; h32++) {      // hs32 + h32 = h
-                    for (int w32 = 0; w32 < 32; w32++) {  // ws32 + w32 = w
+        for (int hs32 = 0; hs32 < H; hs32 += tile_H) {        // iterate over h with stride 32
```

Comment:
nit: these variables are now confusing and the comments about stride 32 no longer accurate. Might be worthwhile updating both.